scripts/anvil_tools/batch_upsert_entities_standard.py

```diff
@@ -55,6 +55,30 @@ def convert_string_to_list(input_string):
     return output_list
 
 
+def create_list_attr_operation(var_attribute_list_name):
+    """Return json request for a single operation to create an attribute of type array/list."""
+
+    return '{"op":"CreateAttributeValueList","attributeName":"' + var_attribute_list_name + '"},'
+
+
+def add_list_member_operation(var_attribute_list_name, var_attribute_list_member):
+    """Return json request for a single operation to add a list member to an attribute of type array/list."""
+
+    return '{"op":"AddListMember","attributeListName":"' + var_attribute_list_name + '", "newMember":"' + var_attribute_list_member + '"},'
+
+
+def create_non_array_attr_operation(var_attribute_name, var_attribute_value):
+    """Return json request for a single operation to create a non-array attribute."""
+
+    return '{"op":"AddUpdateAttribute","attributeName":"' + var_attribute_name + '", "addUpdateAttribute":"' + var_attribute_value + '"},'
+
+
+def create_single_entity_request(var_entity_id, var_entity_type, single_entity_operations_list):
```

Comment:
is the list input an actual list, or a string that looks like a list? if it's an actual list object, i'm not sure how that will behave with this string interpolation? if it's not a list object, i'd clarify what this input should look like in the function description comment